src/cljs/athens/style.cljs

```diff
@@ -1,59 +1,13 @@
 (ns athens.style
   (:require
     [athens.config :as config]
+    [athens.theme :refer [THEME-DARK THEME-LIGHT]]
     [athens.util :as util]
     [garden.color :refer [opacify hex->hsl]]
     [stylefy.core :as stylefy]
     [stylefy.reagent :as stylefy-reagent]))
 
 
-(def THEME-DARK
-  {:link-color          "#2399E7"
-   :highlight-color     "#FBBE63"
-   :text-highlight-color "#FBBE63"
-   :warning-color       "#DE3C21"
-   :confirmation-color  "#189E36"
-   :header-text-color   "#BABABA"
-   :body-text-color     "#AAA"
-   :border-color        "hsla(32, 81%, 90%, 0.08)"
-   :background-minus-1  "#151515"
-   :background-minus-2  "#111"
-   :background-color    "#1A1A1A"
-   :background-plus-1   "#222"
-   :background-plus-2   "#333"
-
-   :graph-control-bg    "#272727"
-   :graph-control-color "white"
-   :graph-node-normal   "#909090"
-   :graph-node-hlt      "#FBBE63"
-   :graph-link-normal   "#323232"
-
-   :error-color         "#fd5243"})
-
-
-(def THEME-LIGHT
-  {:link-color          "#0075E1"
-   :highlight-color     "#F9A132"
-   :text-highlight-color "#ffdb8a"
-   :warning-color       "#D20000"
-   :confirmation-color  "#009E23"
-   :header-text-color   "#322F38"
-   :body-text-color     "#433F38"
-   :border-color        "hsla(32, 81%, 10%, 0.08)"
-   :background-plus-2   "#fff"
-   :background-plus-1   "#fbfbfb"
-   :background-color    "#F6F6F6"
-   :background-minus-1  "#FAF8F6"
-   :background-minus-2  "#EFEDEB"
-   :graph-control-bg    "#f9f9f9"
-   :graph-control-color "black"
-   :graph-node-normal   "#909090"
-   :graph-node-hlt      "#0075E1"
-   :graph-link-normal   "#cfcfcf"
-
-   :error-color         "#fd5243"})
-
-
```

Comment:
Do we like having a separate theme file @shanberg ?

Reply:
One main thing is keeping the find-in-page compiled file small. Hence separated it out. Separating it out to a file seemed like a decent abstraction

Reply:
- How meaningful is the size difference?
- Do we get any reduction/minification for free with tree-shaking?

All else being equal, it’s easier to manage the styles when they're all in the same place, so I'm trying to understand the tradeoff.

Reply:
> How meaningful is the size difference?

~35MB vs ~12MB. It makes sense  because config, deps and all the stuff that's needed from athens style will get compiled. 

> tree shaking

style uses all these deps one way or another. Afaiu this won't help

Reply:
Any more suggestions here? mark it resolved?